### ISSUE TITLE
XP trading

### ISSUE THREAD

**Issue by wookiefever** [RegalOwl/HyperConomy-Documentation#4]
Hi I am not sure if this is an issue or what this would be classified as but I thought I would post here for some clarification.

When I tried to set up XP trading through signs:
experience

[sell:buy]
1

the sign does not register. I can sell xp through command but not through signs... is this a bug?

**Comment by RegalOwl** [comment]
Sounds like a bug.  I'll take a look.

**Comment by wookiefever** [comment]
I don't know what is going on. I logged into the game later and XP trading was working. Then I logged in today and I cannot use signs to buy or sell anything but a friend can.. I am getting ready to uninstall hyperconomy because it is causing so many headaches

**Comment by RegalOwl** [comment]
You're using transaction signs?  Are you holding anything?  It might interfere with the clicking.

**Comment by wookiefever** [comment]
I am using signs. I have tried using signs while having something in my hand and not having something there. They used to work for me and not my friend. Now they are working for her but not me. I am not sure what is going on really.

**Comment by RegalOwl** [comment]
You're sure you're clicking on a transaction sign and not an info sign?  Transaction signs have the brackets []  You should have [Sell:Buy] on the sign for it to work properly,

**Comment by wookiefever** [comment]
Yup I am positive. I will log in to see if it will work for me today.
